data/hai/unfettered side missions.txt

```diff
@@ -147,6 +147,127 @@ mission "Unfettered: Untrustable"
 
 
 
+mission "<uhai> 5 Annex: Carrier Analysis"
+	minor
+	invisible
+	landing
+	source "Firelode"
+	to offer
+		has "<uhai> 5: Auxiliary Scans: done"
+		has "ship model: Carrier"
+		random < 30
+		not "unfettered: outcast"
+	on offer
+		conversation
+			`As you finish the landing procedure you receive a warning from one of your first mates: a Lightning Bug is headed straight towards his Carrier, at full speed!`
+				to display
+					not "flagship model: Carrier"
+			`As you finish the landing procedure you a warning siren triggers on the bridge. One of your officers warns you that a Lightning Bug is headed straight towards you, at full speed!`
+				to display
+					has "flagship model: Carrier"
+			`	Fortunately it stops as soon as it enters scanning range, and begins a full sweep of your ship, before hailing you. You immediately recognize Mera. "Hey there, <first>! I was warned about a Carrier arriving," she looks to her side, probably at one member of her crew as she continues, "because someone thought it was the Giftbringer..." before looking back to you, with a smile, "Anyway, what a pleasant surprise to know it was you. I hope you dont mind me taking some scans, you'll be compensated for it naturally. In fact, I'll allow you to come with me to the lab and choose any item you desire!"`
+			choice
+				`	"That seems like a good deal, I'm in!"`
+					goto deal
+				`	"You can do better than scan that ship, in fact I'm willing to sell it to you."`
```

Comment:
```suggestion
				`	"Would you like to buy it instead?"`
```
Idk. Something like this could be better. ~~Also, it avoids another 'in fact'.~~